### PR TITLE
Refactor project structure

### PR DESCRIPTION
**Pull request by yebenes**

### CODE REVIEW

File: src/app/editor/editor.component.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        if (!cObject.root.get('doc-title')) {
+          cObject.root.put('doc-title', cObject.createString('New document'));
+        }
+
+        // Open the doc in the editor
+        this._title = cObject.root.get('doc-title');
+        this.editor.edit(cObject.root.get('doc'));
+
+        this.editor.onSelectionChanged((annotations) => {
+          this.annotations = annotations;
```
Comment (llopv) [inline src/app/editor/editor.component.ts]: To adapt the code to recent SwellRT API changes, you should change these two lines by:
```js
this.editor.onSelectionChanged((range.annotations) => {
  this.annotations = range.annotations;
```

**Comment by llopv** [comment]
Hey guys!

Can you check this error?

captura de pantalla de 2016-08-17 22-03-04

**Comment by llopv** [comment]
And seems that snackbar is not loaded:

captura de pantalla de 2016-08-17 22-12-25

File: src/app/editor/editor.component.ts (hunk 1 above)
Comment (yebenes) [inline src/app/editor/editor.component.ts]: Perfect! :)

**Comment by yebenes** [comment]
The error is a local error of your computer (on my computer also happened). You simply have to change the type of variable $

:)

**Comment by yebenes** [comment]
I'm going to review the snackbar bar problem :/

**Comment by yebenes** [comment]
Snackbar fixed! :dancers:

**Pull request merged**